### ISSUE TITLE
auto-updater suggests old version

### ISSUE THREAD

**Issue by sschlesier** [git-for-windows/git#1955]
### Setup

Two days in a row the git auto-updater has posted a windows notification suggesting that I upgrade to 2.19.1.  On the first day I blindly clicked yes, and the installer then pointed out I already had 2.19.2 installed.  I cancelled the install process.  Today the same upgrade to 2.19.1 popped up again, I simply clicked no this time.

It seems the auto-updater is not correctly informed of the installed version.

```
$ git --version --build-options
git version 2.19.2.windows.1
cpu: x86_64
built from commit: 26dcaa1b6b5fd862db3ec40983e33ff3432f1166
sizeof-long: 4
sizeof-size_t: 8

```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

```
Microsoft Windows [Version 10.0.17134.376]
```

 - What options did you set as part of the installation? Or did you choose the
   defaults?

```
Editor Option: VIM
Custom Editor Path:
Path Option: CmdTools
SSH Option: OpenSSH
CURL Option: WinSSL
CRLF Option: CRLFAlways
Bash Terminal Option: MinTTY
Performance Tweaks FSCache: Enabled
Use Credential Manager: Enabled
Enable Symlinks: Enabled
```

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

I've been using the auto-updater since it was released.  I only install stable builds.  This machine is about 4 years old.  Before the auto-updater I tried to install the latest builds manually as I noticed they were available.  This machine has had most, but probably not all of the stable release in the past 4 years installed on it.

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

NA

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
NA
```

**Comment by michaelblyons** [comment]
Duplicate of #1953. The 2.19.2 build was downgraded to a pre-release after some people had already installed it.

**Comment by sschlesier** [comment]
I'm sorry I didn't find that issue.  I searched auto-update and a few variations.  I should have searched on version numbers.

I wonder if "failing forward" and publishing a 2.19.2(2) or 2.19.3 which was just really the last good release with a new version and release notes would have been a smoother way to remove the bad 2.19.2.

**Comment by dscho** [comment]
> I wonder if "failing forward" and publishing a 2.19.2(2) or 2.19.3 which was just really the last good release with a new version and release notes would have been a smoother way to remove the bad 2.19.2.

It is always hard to make such judgement calls. I was very close to releasing v2.19.2(2), but there was a pretty big change in the meantime that could potentially cause even more mayhem than the broken 32-bit Git Bash (or the "downgrade" to v2.19.1): we now use OpenSSL v1.1.1. And I really need a few more days to build confidence that that upgrade works. For technical reasons, it would have been really, really hard to revert that OpenSSL upgrade temporarily just to release a v2.19.2(2).

Honestly, if it hadn't been for that OpenSSL issue, I *would* have released v2.19.2(2) in one big hurry.
